server/dispatch.yaml

```diff
@@ -0,0 +1,5 @@
+dispatch:
+  - url: "app-proxy.testing.passculture.team/*"
```

Comment:
TODO: automate with deploy script when https://passculture.atlassian.net/browse/PC-14035 is DONE

TODO: move to appropriate repository (see with @mmachado-pass)